app/src/main/res/values/strings.xml

```diff
@@ -529,7 +529,7 @@
     <string name="auto">Auto</string>
     <string name="switch_view">Switch View</string>
     <!-- App update notification -->
-    <string name="app_update_notification_content_title">NewPipe Update Available!</string>
+    <string name="app_update_notification_content_title">Install the fresh NewPipe version.</string>
```

Comment:
Please revert this change

Reply:
You got it, but it is not a direct instruction like elsewhere. It is also the type of tiring stuff one sees a lot of in Windows.